projects/plugins/jetpack/enhanced-open-graph.php

```diff
@@ -139,7 +139,7 @@ function enhanced_og_video( $tags ) {
 			$secure_video_url = 'https://www.youtube.com/embed/' . $id;
 		} elseif ( strstr( $video_url, 'vimeo' ) ) {
 			preg_match( '|vimeo\.com/(\d+)/?$|i', $video_url, $match );
-			$id               = (int) $match[1];
+			$id               = isset( $match[1] ) ? (int) $match[1] : null;
```

Comment:
Making sure $match[1] is set.

Reply:
I have restored the original code because the errors we see have nothing to do with the code itself. It seems Vimeo embed links look and/or behave differently these days so dealing with that will go to a different PR after some better research.